sdk/src/test/java/com/mapbox/maps/ViewAnnotationManagerTest.kt

```diff
@@ -305,6 +311,28 @@ class ViewAnnotationManagerTest {
     assert(viewAnnotationManager.viewUpdatedListenerSet.isEmpty())
   }
 
+  @Test
+  fun defaultViewAnnotationUpdateMode() {
+    every { renderer.viewAnnotationMode } returns ViewAnnotationManager.DEFAULT_UPDATE_MODE
+    assertEquals(
+      ViewAnnotationManager.DEFAULT_UPDATE_MODE,
+      viewAnnotationManager.getViewAnnotationUpdateMode()
+    )
+  }
+
+  @Test
+  fun setViewAnnotationUpdateMode() {
+    viewAnnotationManager.setViewAnnotationUpdateMode(ViewAnnotationUpdateMode.MAP_FIXED_DELAY)
+    verify(exactly = 1) { renderer.viewAnnotationMode = ViewAnnotationUpdateMode.MAP_FIXED_DELAY }
+  }
+
+  @Test
+  fun getViewAnnotationUpdateMode() {
+    viewAnnotationManager.setViewAnnotationUpdateMode(ViewAnnotationUpdateMode.MAP_FIXED_DELAY)
```

Comment:
not fully understand why did you call `setViewAnnotationUpdateMode` while checking getter

Reply:
thanks, revisited